src/gui/tray/UserModel.cpp

```diff
@@ -40,7 +40,7 @@ User::User(AccountStatePtr &account, const bool &isCurrent, QObject *parent)
         this, &User::slotRefresh);
 
     connect(_account.data(), &AccountState::stateChanged,
-            [=]() { if (isConnected()) {slotRefresh();} });
+            [=]() { if (isConnected()) {slotRefreshImmediately();} });
```

Comment:
Unsure so I'll ask: won't that risk causing issues when there's no push notification support?

Or I guess worst case it means it asks twice at very close interval after a connect when we're still polling?

Reply:
@er-vin `slotRefreshImmediately()` is virtually the same as slotRefresh(). The former just doesn't utilize timers to reduce the frequency of requests. I don't see any risks, as going online happens before WS is connected and ready, so, before this happens we need to have some data in the model, otherwise we'd end up having an empty Tray window (with no activities/notifications).

Reply:
OK, fair enough. This is what I thought, but I preferred to double check with you.